lib/bgs/student_school.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module BGS
+  class StudentSchool
+    def initialize(proc_id:, vnp_participant_id:, payload:, user:)
+      @user = user
+      @proc_id = proc_id
+      @vnp_participant_id = vnp_participant_id
+      @dependents_application = payload['dependents_application']
+    end
+
+    def create
+      bgs_service.create_child_school(submit_params(child_school))
+      bgs_service.create_child_student(submit_params(child_student))
+    end
+
+    private
+
+    def child_school
+      @child_student = BGSDependents::ChildSchool.new(
+        @dependents_application
+      )
+    end
+
+    def child_student
+      @child_student = BGSDependents::ChildStudent.new(
+        @dependents_application
+      )
+    end
+
+    def submit_params(child_object)
+      {
+        vnp_proc_id: @proc_id,
+        vnp_ptcpnt_id: @vnp_participant_id
+      }.merge(child_object.params_for_686c)
+    end
+
+    def bgs_service
+      @service ||= BGS::Service.new(@user)
```

Comment:
Is `BGS::Services` & `BGS::Service` different? I guess this is really a question about the BGS client lib itself.

Reply:
It is different. The gem changed names at not too long ago. I was hesitant to change naming conventions I saw in the lib folder. This is confusing though. I suppose we could rename this folder to something like `form_686_submission` so it would read like `Form686Submission::Service.new(@user)` 🤔


https://github.com/department-of-veterans-affairs/vets-api/pull/4445

WDYT?